backend/views/users.py

```diff
@@ -0,0 +1,32 @@
+from flask import current_app as app
+from flask_restful import Resource, Api, fields, marshal_with, reqparse
+
+from views.blueprints import users_bp
+from repos.users import UsersRepo
+
+users_api = Api(users_bp)
+
+user_api_model = {
```

Comment:
What is the purpose of this? I cant find its usage anywhere.

Reply:
For user_api? Or user_api_model?